cocos/2d/CCDrawNode.h

```diff
@@ -41,6 +41,28 @@ NS_CC_BEGIN
 
 static const int DEFAULT_LINE_WIDTH = 2;
 
+class Triangulate
```

Comment:
I think it is better to add the url of the codes comes from.

Reply:
The url is http://www.flipcode.com/archives/Efficient_Polygon_Triangulation.shtml. 
I wrote the url in the first comment up there. Do you mean that I need write the url into the source code's comment ?

Reply:
Yep, i think it is better.

Reply:
Ok, I've added in the new commit.